packages/kit/test/nodash/internal.test.ts

```diff
@@ -91,4 +91,12 @@ describe('nodash internal', () => {
       expect(_.upperFirst('abc')).to.equal('Abc');
     });
   });
+
+  describe('format', () => {
+    it('returns a title case string given a camel case one', () => {
```

Comment:
@maggiben The input to this test already has first character as upper case. Could you add another test case that supplies a string with lowercase first character?

Also, what happens if the string is already formatted in title case?

Reply:
@peternhale I added a few more tests the original function comes from another repo: https://github.com/salesforcecli/plugin-org/blob/main/src/shared/utils.ts